### ISSUE TITLE
External Links

### ISSUE THREAD

**Issue by aivorra** [pluginsGLPI/genericobject#94]
Hi, I'm using this plugin and it works great, but I'm trying to add External Links to the "item type" that I built with the plugin and I don't have this option.

How could I add this option/funcion?

Best Regards.

**Comment by aivorra** [comment]
I solved in this way:

1. I added the line 
`$this->addStandardTab('Link', $ong, $options);` 
after 
`if ($this->canUseTickets()) {
            ...
          }` 
into the file "genericobject/inc/object.class.php"

2. I added into the DDBB > Table 'glpi_links_itemtypes' a new reg with this information:
links_id = the numbre of the external link that i want to link.
itemtype = the name of the itemtype (eg. PluginGenericobjectRadio)

3. But I'm still trying to show the External Link on the 'default item to show'
